testsuite/parallel_execution_performance.py

```diff
@@ -5,30 +5,31 @@
 
 import subprocess
 import re
+import platform
 
 # Set the tps and speedup ratio threshold for block size 1k, 10k and 50k
 THRESHOLDS = {
-    "1k_8": 15000,
-    "1k_16": 19000,
-    "1k_32": 20000,
-    "10k_8": 27000,
-    "10k_16": 46000,
-    "10k_32": 63000,
-    "50k_8": 29000,
-    "50k_16": 53000,
-    "50k_32": 82000,
+    "1k_8": 11000,
```

Comment:
Please note that these thresholds have changed because of the workload pattern has changed. Earlier, the account working set size was just 10k and we used to regenerate the account for every block. This is not really reflective of the workload pattern we will see. So instead, I have increased the working set size to be 200k and keeping the same working set across the blocks, so potentially they might be re-used across blocks.